etna/insights/models.py

```diff
@@ -18,9 +23,6 @@ class InsightsIndexPage(TeaserImageMixin, BasePage):
 
     sub_heading = models.CharField(max_length=200, blank=False)
     body = StreamField(InsightsIndexPageStreamBlock, blank=True, null=True)
-    featured_insight = models.ForeignKey(
-        "insights.InsightsPage", blank=True, null=True, on_delete=models.SET_NULL
-    )
```

Comment:
You appear to have removed a field here that was added recently. This will need to be reverted.

Reply:
Yes , sorry about that, again merge issues.